Bio/AlignIO/EmbossIO.py

```diff
@@ -57,8 +57,6 @@ def write_alignment(self, alignment):
         handle.write("#\n")
         handle.write("#=======================================\n")
         handle.write("\n")
-        # ...
-        assert False
```

Comment:
In general ``assert False`` is something better written as ``raise ValueError`` or similar. In this specific case, as this is a base object which should have these methods redefined, ``raise NotImplementedError`` as in the example here: https://github.com/biopython/biopython/blob/biopython-173/Bio/SeqIO/Interfaces.py#L54